scripts/variantstore/variant-annotations-table/GvsMapDroppedDuplicateVIDs.wdl

```
@@ -0,0 +1,176 @@
+        #    output to this `to_search.vcf` file.
+        bash bcftools_commands.sh
+
+        # 8. The generated bcftools commands will likely have overlapping ranges and generate duplicate and non-sequential entries
+        #    in the `to_search.vcf` file. Fix that by sorting and deduplicating the VCF.
+        bcftools sort to_search.vcf | bcftools norm -d none -o to_search.sort.dedup.vcf
+
+        # 9. Make a normalized (left aligned) version of this VCF.
+        bcftools norm -f ~{reference.reference_fasta} to_search.sort.dedup.vcf > to_search.sort.dedup.norm.vcf
+
+        # 10. At this point the normalized VCF should contain all the duplicate vids, but because the bcftools searches are written
+        #     to search a range of positions, they will likely also match some variants that do not correspond to duplicate vids.
+        #     Run the following to clean out non-duplicate vid entries from this file.
+        python /app/filter_vcf_by_vids.py dropped_duplicate_vids.tsv to_search.sort.dedup.norm.vcf > hits_only.vcf
+
+        # 11. Now we can correlate the entries in this `hits_only.vcf` file back to the non-left aligned version that uses the same
+        #     positions as GVS.
+        python /app/map_input_alignments_to_left_alignments.py to_search.sort.dedup.vcf hits_only.vcf > dropped_duplicate_mappings.tsv
+
+        # 12. Load the duplicate vid mapping into BigQuery
+        bq load --project_id ~{project} --source_format=CSV --skip_leading_rows=1 --field_delimiter="\t" \
+            ~{dataset}.~{duplicate_mapping_table_name} dropped_duplicate_mappings.tsv \
+            vid:STRING,chr:STRING,input_location:INTEGER,input_position:INTEGER,input_ref:STRING,input_alt:STRING,left_aligned_location:INTEGER,left_aligned_position:INTEGER,left_aligned_ref:STRING,left_aligned_alt:STRING,info_field:STRING
+
+        # 13. Remove existing mappings for these duplicates from the mapping table
+        #
+        # bq query --max_rows check: ok delete
```

What is this commented out bq query here for?

---

This is carrying on the pattern of auditing `bq query`s  from VS-1396 motivated by that unfortunate incident in Echo where we silently truncated output to 100 results. This is just saying that this query has been audited and is okay because it's not returning results, only performing a delete.